README.md

```diff
@@ -150,16 +157,17 @@ by adding -DUSE_CPU_ONLY=1 to the cmake command line.
 
 # Contributing
 
-For Windows, you can use a release package, see ["I want to help"](#windows).
+- For Windows, you can use a release package, see ["I want to help"](#windows).
 
-Unix and macOS, after finishing the compile and while in the build directory:
+- For Unix and macOS, after finishing the compile and while in the build directory:
 
-    # Copy leelaz binary to autogtp subdirectory
-    cp leelaz autogtp
+      # Copy leelaz binary to build/autogtp/ subdirectory
+      cp leelaz autogtp
 
-    # Run AutoGTP to start contributing
-    ./autogtp/autogtp
+      # Run AutoGTP to start contributing
+      ./autogtp/autogtp
 
+For more details about autogtp or for more advanced use, you can see [the autogtp readme](https://github.com/gcp/leela-zero/blob/next/autogtp/README.md)
```

Comment:
Reword: autogtp -> AutoGTP  and ... you can see ... ->... please see ...

Can you make the link relative somehow?

Reply:
it was actually surprisingly easy, but thank god the relative links now works ! 
github is too smart

`[relativelink](/autogtp/README.md)`

see my branches, ctrl +f "relativelink"
- master : https://github.com/wonderingabout/leela-zero/tree/master
- next : https://github.com/wonderingabout/leela-zero/tree/next

when the pull request is merged i'll remove these commits